shell/browser/notifications/win/win32_notification.cc

```diff
@@ -49,13 +50,15 @@ void Win32Notification::Show(const NotificationOptions& options) {
     existing->tag_.clear();
 
     this->notification_ref_ = std::move(existing->notification_ref_);
-    this->notification_ref_.Set(options.title, options.msg, image);
+    this->notification_ref_.Set(base::UTF16ToWide(options.title),
+                                base::UTF16ToWide(options.msg), image);
```

Comment:
same here, should `NotificationOptions::{title, subtitle, msg, ...}` all be `std::wstring` instead of `u16string`?

Reply:
The blink structures we're talking to are using u16string, e.g. https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/public/common/notifications/platform_notification_data.h;l=38;drc=184bf7b03160ebf2217ab4711118d013a750908a. 

IIUC, **on Windows** the underlying data between wstring and u16string is interchangeable on Windows. Are there any guidelines on when to prefer one over the other? (Or are the differences just due to the large upstream codebase & large number of programmers?)